sdk/samples/iot/hub/linux/how_to_iot_hub_samples_linux.md

```diff
@@ -273,12 +271,12 @@ _The following was run on an Ubuntu Desktop 18.04 environment._
     - Go to your Azure IoT hub page.
     - Click on "IoT devices" under "Explorers".
     - From the list of devices, click on your device (created on step 8).
-      ![Device page](./resources/embc_samples_01_device.png)
+      ![Device page](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/docs/iot/resources/embc_samples_01_device.png)
```

Comment:
these should be relative to get them to resolve in the document (and not just be a link):
```suggestion
      ![Device page]../../../../docs/iot/resources/embc_samples_01_device.png)
```